app/components/forms/wizard/custom-form-input.hbs

```diff
@@ -6,16 +6,36 @@
   </span>
 </div>
 <div {{did-update this.updated @field}} class="ui action input" style="width: inherit;">
-  <Input type="text" placeholder="Field Name" @value={{this.name}} />
+  <Input type="text" id="shortField" placeholder="Field Name" @value={{this.name}} style="display: none;" />
+  <Textarea cols="60" rows="1" id="paraField" style="width: 100%; overflow: hidden; display: block;" placeholder="Field Name" oninput="auto_height(this)" @value={{this.name}}></Textarea>
```

Comment:
The textarea should be shown at the order page if the custom field is selected as long answer.